Templates/MonoGame.Templates.VSExtension/Properties/AssemblyInfo.cs

```diff
@@ -2,32 +2,7 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-// General Information about an assembly is controlled through the following 
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
-[assembly: AssemblyTitle("MonoGame.Templates.VSExtension")]
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("MonoGame.Templates.VSExtension")]
-[assembly: AssemblyCopyright("")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
-
 // Setting ComVisible to false makes the types in this assembly not visible 
 // to COM components.  If you need to access a type in this assembly from 
 // COM, set the ComVisible attribute to true on that type.
-[assembly: ComVisible(false)]
-
-// Version information for an assembly consists of the following four values:
-//
-//      Major Version
-//      Minor Version 
-//      Build Number
-//      Revision
-//
-// You can specify all the values or you can default the Build and Revision Numbers 
-// by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
```

Comment:
@dellis1972 I had to delete these attributes, because there is an MSBuild auto-generating something similar, that was causing build conflicts. Is this by design? Also where is that MSBuild pulling in it's information, because we should probably update `MonoGame Team` to `MonoGame Foundation`

Reply:
You can delete the AssemblyInfo.cs , its all set in MSBuold properties now https://learn.microsoft.com/en-us/dotnet/standard/assembly/set-attributes-project-file

Reply:
Done.